RAM_Adapter/Create/Level.cs

```diff
@@ -53,120 +53,179 @@ public partial class RAMAdapter
         /**** Private methods                           ****/
         /***************************************************/
 
+        //private bool CreateCollection(IEnumerable<Level> bhomLevels)
+        //{
+        //    if (bhomLevels.Count() != 0)
+        //    {
+        //        //sort levels by elevation
+        //        IOrderedEnumerable<Level> orderedBhomLevels = bhomLevels.OrderBy(o => o.Elevation);
+        //        List<Level> sortedBhomLevels = new List<Level>();
+
+        //        //Check levels for negatives
+        //        if (orderedBhomLevels.First().Elevation < 0)
+        //        {
+        //            throw new Exception("Base level can not be negative for RAM. Please move model origin point to set all geometry and levels at 0 or greater.");
+        //        }
+
+        //        //Check levels for base level = 0, remove if occurs
+        //        if (orderedBhomLevels.First().Elevation == 0)
+        //        {
+        //            sortedBhomLevels = orderedBhomLevels.Where(level => level.Elevation != 0).ToList();
+        //        }
+        //        else
+        //        {
+        //            sortedBhomLevels = orderedBhomLevels.Where(level => level.Elevation != 0).ToList();
+        //        }
```

Comment:
Is this if statement necessary? Looks like both the same line or code whether or not the if condition is met

Reply:
Thanks for comments here @LukeBastianBH . Although I do agree that the if statement seem unnecessary, and on top of that is commented out (something we generally try to avoid in the BHoM as old code can be found via git-history), this particular PR should not really be scope for an in-depth code review, but just functionality.

The particular code was merged in https://github.com/BHoM/RAM_Toolkit/pull/198 which was the appropriate time to catch this. 

To fix this, you would need to raise a separate compliance/bug issue and PR that then needs full code as well as functionality review. I am happy for this to be done, but requires someone to do it, and another individual to review.

the other option (that I would recommend in this case) is to raise an issue for it, and then leave it be, and just ensure that the functionality of the toolkit in the beta candidate installer is working as intended.

TL;DR: Do not need to review the code on this PR unless something breaks. Only testing required is functionality.